generated/1.30/apis/go.mod

```diff
@@ -3,7 +3,7 @@ module go.pinniped.dev/generated/1.30/apis
 
 go 1.22.0
 
-toolchain go1.23.3
+toolchain go1.23.2
```

Comment:
It looks like Pinny is trying to upgrade Go to 1.23.3, so why downgrade it back?

Reply:
Maybe you were trying to make the verify-codegen CI job pass? I just bumped Go in CI (see comment below). Hopefully that will do the trick.